pkg/koordlet/metrics/common.go

```diff
@@ -86,3 +114,45 @@ func RecordBESuppressCores(suppressType string, value float64) {
 	labels[BESuppressTypeKey] = suppressType
 	BESuppressCPU.With(labels).Set(value)
 }
+
+func RecordContainerCPI(containerID, podUID string, value float64) {
```

Comment:
should add more labels to pod/container metrics: pod namespace/name, container name